mne/forward/forward.py

```diff
@@ -967,48 +968,104 @@ def _select_orient_forward(forward, info, noise_cov=None, copy=True):
     return forward, info_picked
 
 
+def _triage_loose(src, loose, fixed='auto'):
+    _validate_type(loose, (str, dict, 'numeric'), 'loose')
+    _validate_type(fixed, (str, bool), 'fixed')
+    orig_loose = loose
+    if isinstance(loose, str):
+        _check_option('loose', loose, ('auto',))
+        if fixed is True:
+            loose = 0.
+        else:  # False or auto
+            loose = 0.2 if src.kind == 'surface' else 1.
+    src_types = set(_src_kind_dict[s['type']] for s in src)
+    if not isinstance(loose, dict):
+        loose = float(loose)
+        loose = {key: loose for key in src_types}
+    loose_keys = set(loose.keys())
+    if loose_keys != src_types:
+        raise ValueError(
+            f'loose, if dict, must have keys {sorted(src_types)} to match the '
+            f'source space, got {sorted(loose_keys)}')
+    # if fixed is auto it can be ignored, if it's False it can be ignored,
+    # only really need to care about fixed=True
+    if fixed is True:
```

Comment:
we agree that fixed only applies to the surface?

Reply:
surface+discrete -- that's how it is in `master` and I agree we should only allow it for those (not volume)